test/examples/ucp_hello_world.c

```diff
@@ -329,6 +332,14 @@ static int run_ucx_server(ucp_worker_h ucp_worker)
         goto err;
     }
 
+    test_str = (char*) calloc(test_string_length+1, sizeof(char));
+    if (!test_str) {
+        printf("unable to allocate memory\n");
+        goto err_ep;
+    }
+
+    generate_random_string(test_str, test_string_length);
```

Comment:
str[size] contains 0 after the call. So, one byte allocated as extra space before.
Perhaps, generate_random_string(test_str, test_string_length + 1);